po/fr.po

```diff
@@ -66,6 +70,10 @@ msgid ""
 "That means history, cookies, local storage, passwords, etc. are all blown "
 "away as soon as you leave."
 msgstr ""
+"1   Toujours incognito. Dès que vous ouvrez une fenêtre Ephemeral et jusqu'à "
```

Comment:
The original string for this (and the following three translations) contain an emoji number; you should add it to your translation as well